plugin/LaserInputGeometry.h

```diff
@@ -96,7 +115,14 @@ class LaserInputGeometry : public RenderedGeometry
     bufferArray.appendInterleavedBufferLayout(BufferLayout("weight", 1, BufferComponentType::uint32));
   }
 
-  size_t numLines;
+  size_t numPoints, numLines;
+
+  void configure(LaserGeometryShader& geometryShader, const GeometryMask& mask, float maskThreshold, size_t weight)
+  {
+    geometryShader.getMask().set(&mask);
+    geometryShader.getMaskTreshold.set(maskThreshold);
```

Comment:
`getMaskThreshold` doesn't seem to be a member or field of `LaserInputGeometry`?

Reply:
idem